src/neo/Ledger/Blockchain.cs

```diff
@@ -304,6 +304,7 @@ private void OnInventory(IInventory inventory, bool relay = true)
             };
             if (relay && rr.Result == VerifyResult.Succeed)
                 system.LocalNode.Tell(new LocalNode.RelayDirectly { Inventory = inventory });
+            Sender.Tell(rr);
             Context.System.EventStream.Publish(rr);
```

Comment:
`Context.System.EventStream.Publish(rr);`  we can remove it now?

Reply:
No. Why do we remove it?

Reply:
I thought we were using EventStream for SyncManager, and if we use `Sender.Tell(rr)`, it can still get the VerifyResult.

Reply:
If you do not subscribe to the event, then you will not receive messages from `EventStream`. So `SyncManager` will not receive duplicate `RelayResult`s.

Reply:
I probably not understand the `EventStream` usage scenario, since `SyncManager` can get the `VerifyResult` by `Sender.Tell` now.